### PR TITLE
Media browser tweaks

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

UI Tweaks for the media browser:

- Show artwork in list if the artworks differ
- Scroll to top on navigate
- Adjust font size

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by zsarnett — APPROVED** [review]

**Comment by zsarnett** [comment]
Something maybe out of this PR. @cgtobi requested that when he is in a playlist that has a lot of music/content he has to scroll all the way back to to the top in order to go back a folder. Would be cool to be able to shrink the header on scroll. Problem is that the browser is not the element with the scroll bar but the dialog is. So we would need to some way make it so the height comes from the dialog and then 100% height on the browser and scroll within there. idk

**Comment by bramkragten** [comment]
That means the header should be part of the dialog and not of the browser, would be nice for a different PR

**Comment by bramkragten** [comment]
Add a back button to the header of the dialog now:

image

**Comment by zsarnett** [comment]
hmm, I think we need the functionality. It is a bit weird. The issue is we should build the browser not solely based on being in a dialog so removing the back within it may not be the best. I would just keep this for now. Maybe we can rewrite it a bit to cover cases where the browser takes up 100% of the height its given and then we scroll within the browser element itself.

**Comment by ludeeus** [comment]
That looks weird, can we add something to the rendered content to stop it from rendering the second back button if in a dialog?

**Comment by cgtobi** [comment]
Agree, two back buttons look wired. Especially since they look differentially labeled and would be confusing. Which does which?

**Comment by bramkragten** [comment]
image
image
image

**Comment by zsarnett** [comment]
8fcdf742ddd275b2478af650d7d782f3

**Comment by zsarnett** [comment]
<a href="https://gyazo.com/8fcdf742ddd275b2478af650d7d782f3"><video alt="Video from Gyazo" width="720" autoplay muted loop playsinline controls><source src="https://i.gyazo.com/8fcdf742ddd275b2478af650d7d782f3.mp4" type="video/mp4" /></video></a>

**Comment by zsarnett** [comment]
[Image from Gyazo](https://gyazo.com/8fcdf742ddd275b2478af650d7d782f3)

Very poor GIF :)

File: src/components/media-player/dialog-media-player-browse.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import type {
 } from "../../data/media-player";
 import { haStyleDialog } from "../../resources/styles";
 import type { HomeAssistant } from "../../types";
-import { createCloseHeading } from "../ha-dialog";
+import "../ha-header-bar";
```
Comment (ludeeus) [inline src/components/media-player/dialog-media-player-browse.ts:18]:
```suggestion
import "./../ha-dialog";
import "../ha-header-bar";
```

File: src/components/media-player/dialog-media-player-browse.ts [hunk 2]
```diff
@@ -56,18 +56,18 @@ class DialogMediaPlayerBrowse extends LitElement {
         scrimClickAction
         escapeKeyAction
         hideActions
-        .heading=${createCloseHeading(
-          this.hass,
-          this.hass.localize("ui.components.media-browser.media-player-browser")
-        )}
+        flexContent
+        .heading=${false}
```
Comment (ludeeus) [inline src/components/media-player/dialog-media-player-browse.ts:60]:
```suggestion
```

**Review by zsarnett — APPROVED** [review]

**Pull request merged**